src/Bot/Scheduler.cs

```diff
@@ -104,4 +104,61 @@ public static void Dispose()
         if(clock != null)
             clock.Dispose();
     }
+
+    public static uint GetIntervalFromTimes(List<string> scheduleTimes)
+    {
+        scheduleTimes.Sort();
+
+        DateTime now = DateTime.Now;
+		string nowString = now.ToString("HH:mm");
+
+        DateTime nextRestartTimeDT = new DateTime();
+        string nextRestartTime = "";
+
+        if (scheduleTimes.Count == 0) return 999999999;
+        
+		foreach (string time in scheduleTimes)
+		{
+            DateTime timeDT;
+            try 
+	        {	        
+		        timeDT = DateTime.Parse(time);
```

Comment:
Some people might try to type time in `"HH.mm"` format. I didn't see any check for catching it. And when it is parsed, it will be interpreted as day and month so it should be checked. Probably in the discord command function.

Reply:
I changed this to use ParseExact to only accept "HH:mm" (24-hour format) times